flink-streaming-java/src/test/java/org/apache/flink/streaming/api/complex/ComplexIntegrationTest.java

```diff
@@ -214,6 +214,7 @@ public void complexIntegrationTest2() throws Exception {
 
 	@SuppressWarnings("unchecked")
 	@Test
+	@Ignore
```

Comment:
It's called `ComplexIntegrationTest` for a reason :smiley: I would either remove it or add a comment why you ignored the test.